### PR TITLE
Devportal test

### PR DESCRIPTION
**Pull request by SebasGuaquetaRSK**

### CODE REVIEW

File: .github/workflows/devportal-update.yml [hunk 1]
```diff
@@ -0,0 +1,72 @@
+name: Update Devportal Documentation
+
+on:
+  push:
+    paths:
+      - 'README.md'
+    branches:
+      - devportalTest
```
Comment (SebasGuaquetaRSK) [inline .github/workflows/devportal-update.yml:8]:
```suggestion
      - main
```

File: .github/workflows/devportal-update.yml [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+      - 'README.md'
+    branches:
+      - main
+
+jobs:
+  update-docs:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v4
```
Comment (Luisfc68) [inline .github/workflows/devportal-update.yml:15]:
```suggestion
        uses: actions/checkout@eef61447b9ff4aafe5dcd4e0bbf5d482be7e7871 # v4.2.1
```

File: Readme.md [hunk 3]
```diff
@@ -1,33 +1,178 @@
...
+
+## Setup
+To setup the Solidity helper library in your project, run the following npm command:
+
+```bash
+   npm install @rsksmart/btc-transaction-solidity-helper
+```
 ### Future features
+
 These are some features that can increase the library capabilities in the future:
```
Comment (Luisfc68) [inline Readme.md:35]: This section was moved down below (to the line 170) so I think it should be removed from this part

**Review by Luisfc68 — CHANGES_REQUESTED** [review]
I think I won't be able to merge because of these commits
image
maybe you need to squash and sign the result with the account that is verified

File: Readme.md (hunk 3 above)
Comment (SebasGuaquetaRSK) [reply Readme.md:35]: Done

**Comment by SebasGuaquetaRSK** [comment]
I will close this PR in order to open a new one from a new branch without issues on the verification, this due to in my computer I have some issues on the GPG signing (my personal account and the RSK one) so this messed up with the re signing prev commits.

**Pull request closed without merging**